djangocms_url_manager/templates/admin/djangocms_url_manager/url/change_form.html

```diff
@@ -1,4 +1,4 @@
-{% extends "admin/change_form.html" %}
+{% extends is_versioning_enabled|yesno:"djangocms_versioning/admin/mixins/change_form.html,admin/change_form.html" %}
```

Comment:
```suggestion
{% extends is_versioning_enabled|yesno:"djangocms_versioning/admin/mixin/change_form.html,admin/change_form.html" %}
```

This seems to have rolled back to the original with the wrong path for mixins